internal/pkg/manifest/validate_env.go

```diff
@@ -163,3 +164,21 @@ func (c subnetConfiguration) Validate() error {
 func (o environmentObservability) Validate() error {
 	return nil
 }
+
+// Validate returns nil if environmentHTTPConfig is configured correctly.
+func (o environmentHTTPConfig) Validate() error {
+	if err := o.Public.Validate(); err != nil {
+		return fmt.Errorf(`validate "public": %w`, err)
+	}
+	return nil
+}
+
+// Validate returns nil if publicHTTPConfig is configured correctly.
+func (o publicHTTPConfig) Validate() error {
+	for idx, certARN := range o.Certificates {
+		if _, err := arn.Parse(certARN); err != nil {
+			return fmt.Errorf(`validate "certificates[%d]": %w`, idx, err)
```

Comment:
I like `certificates[%d]` which makes the error msg cleaner.
nit:
```suggestion
			return fmt.Errorf(`parse "certificates[%d]": %w`, idx, err)
```

Reply:
I felt like `validate` better describes what we are doing here though. We are just parsing for the sake of validating 🤔  Are you concerned because you don't want user to think that we are validating certs by upserting records? In this case, what do you think of `validate "certificates[%d] arn": %w`?

Reply:
i just feel like the error msg wrapping should describe the behavior. Since we are just parsing the arn i feel like we should wrap the error like this. But it's ok just a nit.